Meshtastic/Helpers/BLEManager.swift

```diff
@@ -883,12 +890,13 @@ class BLEManager: NSObject, CBPeripheralDelegate, MqttClientProxyManagerDelegate
 								}
 							}
 							hopNodes.append(traceRouteHop)
-							routeString += "\(hopNode?.user?.longName ?? (node == 4294967295 ? "Repeater" : String(hopNode?.num.toHex() ?? "unknown".localized))) \(hopNode?.viaMqtt ?? false ? "MQTT" : "") (\(traceRouteHop.snr > 0 ? hopNode?.snr ?? 0.0 : 0.0)dB) --> "
+							routeString += "\(hopNode?.user?.longName ?? String(hopNode?.num.toHex() ?? "unknown".localized)) \(hopNode?.viaMqtt ?? false ? "MQTT" : "") (\(traceRouteHop.snr != -32 ? String(traceRouteHop.snr) : "unknown ".localized)dB) --> "
```

Comment:
nit: this is kinda hard to read
```
routeString += "\(hopNode?.user?.longName ?? String(hopNode?.num.toHex() ?? "unknown".localized)) \(hopNode?.viaMqtt ?? false ? "MQTT" : "") (\(traceRouteHop.snr != -32 ? String(traceRouteHop.snr) : "unknown ".localized)dB) --> "
```

```
let hopName = hopNode?.user?.longName ?? String(hopNode?.num.toHex() ?? "unknown".localized)
let mqttLabel = hopNode?.viaMqtt ?? false ? "MQTT" : ""
let snrLabel = (traceRouteHop.snr != -32 ? String(traceRouteHop.snr) : "unknown ".localized

routeString += "\(hopName) \(mqttLabel) (\snrLabel)dB) --> "
```
^ after refactoring it becomes clear that we have an unnecessary whitespace if mqttLabel is blank.

```
let hopName = hopNode?.user?.longName ?? String(hopNode?.num.toHex() ?? "unknown".localized)
let mqttLabel = hopNode?.viaMqtt ?? false ? "MQTT " : ""
let snrLabel = (traceRouteHop.snr != -32 ? String(traceRouteHop.snr) : "unknown ".localized

routeString += "\(hopName) \(mqttLabel)(\snrLabel)dB) --> "
```
^ fixt `"MQTT "` + `\(hopName) \(mqttLabel)(\snrLabel)`

Reply:
Yes. That's more readable. Will make that update.

Reply:
Done